src/client/views/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<!--
+To change this license header, choose License Headers in Project Properties.
+To change this template file, choose Tools | Templates
+and open the template in the editor.
+-->
+<html ng-app="interview_coding">
+    <head>
+        <% if (title) { %>
+        <title><%= title %></title>
+        <% } %>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+        <!--        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>-->
+        <script src="node_modules/angular/angular.min.js" type="text/javascript"></script>
+        <script src="app/app.js" type="text/javascript"></script>
+        <!--Services-->
+        <script src="app/services/services.js" type="text/javascript"></script>
+        <!--Controllers-->
+        <script src="app/controllers/controllers.js" type="text/javascript"></script>
+        <!--Directives-->
+        <script src="app/directives/directives.js" type="text/javascript"></script>
```

Comment:
No, it isn't, It is better adding on footer all scripts that depends of AngularJS